qemu/tests/create_large_raw_img.py

```diff
@@ -38,11 +40,15 @@ def run(test, params, env):
     try:
         large.create(large.params)
     except TestError as err:
-        if params["err_info"] not in str(err):
+        for info in err_info:
+            if info in six.text_type(err):
+                break
+        else:
             test.fail("CML failed with unexpected output: %s" % err)
     else:
         test.fail("There is no error when creating an image with large size.")
-
-    part.unmount()
-    os.rmdir(mnt_dir)
-    os.remove(loop_img)
+    finally:
+        if part.get_mountpoint():
```

Comment:
we might remove the get_mountpoint() here, for in unmount function we can see:
with MtabLock():
             mountpoint = self.get_mountpoint()
             if not mountpoint:
                 LOG.debug('%s not mounted', self.device)
                 return 1
i.e. unmount will check the mount point.

Reply:
yes, you are right. Updated it, many thanks.